spec/jobs/skill_fetcher_job_spec.rb

```diff
@@ -1,5 +1,62 @@
 require 'rails_helper'
 
 RSpec.describe SkillFetcherJob, type: :job do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe '#fetch_org_info' do
+    let(:make_request) {
+      ->{
+        VCR.use_cassette('github_organization') do
+          GithubClient.new.fetch_org_info('openlexington')
+        end
+      }
+    }
+
+    it 'returns a resource' do
+      expect(make_request.call).to be_a(Sawyer::Resource)
+    end
+
+    it 'contains openlexington as the login' do
+      expect(make_request.call.login).to eq('openlexington')
+    end
+  end
+
+  describe '#fetch_org_member_list' do
+    let(:make_request) {
+      ->{
+        VCR.use_cassette('github_organization') do
+          GithubClient.new.fetch_org_member_list('openlexington')
+        end
+      }
+    }
+
+    it 'returns an array' do
+      expect(make_request.call).to be_a(Array)
+    end
+
+    it 'returns an array of members' do
+      expect(make_request.call.count).not_to eq(0)
+    end
+  end
+
+  describe '#fetch_member_languages' do
+    let(:make_request) {
+      ->{
+        VCR.use_cassette('chaserx') do
+          GithubClient.new.fetch_member_languages('chaserx')
+        end
+      }
+    }
+
+    it 'returns a hash' do
+      expect(make_request.call).to be_a(Hash)
+    end
+
+    it 'returns a hash with language keys and byte values' do
+      expected_hsh = {:Ruby=>969926, :JavaScript=>959973, :CSS=>264713,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Separate every 3 digits in the integer portion of a number with underscores(_).<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.